### ISSUE TITLE
Page templates

### ISSUE THREAD

**Issue by AdamChrimes** [nhsuk/nhsuk-service-manual-backlog#200]
Use this issue to discuss Page templates on the NHS digital service manual.

**Comment by januszjasinski** [comment]
That's right yeah, just out of curiosity is all as we're looking to adopt
this into our workflow

**Comment by henocookie** [comment]
@wa-rren-dev has [submitted a PR](https://github.com/nhsuk/nhsuk-service-manual/pull/1729) to fix this issue! Thanks for spotting @januszjasinski

**Comment by vickytnz** [comment]
This page has inconsistencies with the components that are used - for example this page has the buttons 'back' 
<img width="490" alt="page with back link" src="https://github.com/user-attachments/assets/d6bae3cd-e5e9-454f-b8dd-3305cf4e3883">

whereas the back button in the component has 'go back' https://service-manual.nhs.uk/design-system/components/back-link 

<img width="193" alt="go back" src="https://github.com/user-attachments/assets/341f30a7-f8d9-483c-a275-f2f51df2b3b9">


It would also be good to have an example of the page without the back link. When I try doing this (for example for a success or check your answers page where back can cause some weirdness) , it looks like the header is far too close to the top line

<img width="953" alt="transactional page with the back link removed" src="https://github.com/user-attachments/assets/de25e336-a1cf-4c87-9b91-3f819b984a97">

And this looks different from the general content page 
<img width="958" alt="content page" src="https://github.com/user-attachments/assets/b3ad0b0d-9e73-4b25-9e8e-fa29bdb63b22">

There is some content about reducing the margins but I think that this less about it being a transactional page and more that it might be nested under a back link?
